inference-engine/src/gna_plugin/transformations/broadcast_const.cpp

```diff
@@ -0,0 +1,171 @@
+// Copyright (C) 2021 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include <openvino/cc/ngraph/itt.hpp>
+
+#include "transformations/broadcast_const.hpp"
+
+#include "transformations/utils/transformation_helper.hpp"
+
+#include <ngraph/opsets/opset8.hpp>
+#include <ngraph/pattern/op/wrap_type.hpp>
+#include <ngraph/pattern/op/or.hpp>
+#include "legacy/ngraph_ops/eltwise.hpp"
+#include "ngraph_ops/convolution_ie.hpp"
+#include "ngraph/runtime/reference/broadcast.hpp"
+
+#include <vector>
+#include <functional>
+#include <numeric>
+
+using namespace GNAPluginNS;
+
+NGRAPH_RTTI_DEFINITION(BroadcastConst, "BroadcastConst", 0);
+
+using Node = std::shared_ptr<ngraph::Node>;
+using Nodes = std::vector<Node>;
+using Input = ngraph::Input<ngraph::Node>;
+using Inputs = std::vector<Input>;
+using AxisSet = std::set<size_t>;
+
+namespace {
+
+std::string Shape2Str(const ngraph::Shape & shape) {
```

Comment:
Looks like this is a helper function and can be reused, it is better to move it to shared place, but I think you don't need it at all. ngraph::shape class overloads "<<" to convert shape to ostream.
std::count << ngraph::shape({1, 2, 3});

Reply:
I've already removed that piece of code after another comment